Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -4027,7 +4052,7 @@ public IWebElement LocateElementByLocators(ElementInfo currentPOMElementInfo, IL
                     }
                     else
                     {
-                        elem = LocateElementIfNotAutoLeared(locator, ParentContext, friendlyLocatorElementlist);
+                        elem = LocateElementIfNotAutoLeared(locator, ParentContext??Driver, friendlyLocatorElementlist);
```

Comment:
why we need to add this condition here?

Reply:
if the parent context(which is a web element) is null i.e. could not be found